unity/Assets/StreamingAssets/content/MoM/ck/cotw-tiles/tiles.ini

```diff
@@ -1,143 +1,143 @@
 [TileSideAbandonedShack]
 name={ffg:TILE_ABANDONED_SHACK_MAD09}
-image="{import}/img/Tile_Abandoned_Shack_MAD09.dds"
+image="{import}/img/Tile_Abandoned_Shack_MAD09"
 traits=inside outside medium
 reverse=TileSideHillTop
 
 [TileSideBarn]
 name={ffg:TILE_BARN_MAD09}
-image="{import}/img/Tile_Barn_MAD09.dds"
+image="{import}/img/Tile_Barn_MAD09"
 aspect=1.5
 pps=146.28571
 traits=inside outside big
 reverse=TileSideCoveredBridge
 
 [TileSideCampsite]
 name={ffg:TILE_CAMPSITE_MAD09}
-image="img/Tile_Campsite_MAD09.jpg"
+image="img/Tile_Campsite_MAD09"
 pps=72
 traits=outside small
 reverse=TileSideScarecrow
 
 [TileSideCaveBend]
 name={ffg:TILE_CAVE_BEND_MAD09}
-image="img/Tile_Cave_Bend_MAD09.jpg"
+image="img/Tile_Cave_Bend_MAD09"
 pps=72
 traits=inside small cave
 reverse=TileSideOldOak
 
 [TileSideCaveEntrance]
 name={ffg:TILE_CAVE_ENTRANCE_MAD09}
-image="{import}/img/Tile_Cave_Entrance_MAD09.dds"
+image="{import}/img/Tile_Cave_Entrance_MAD09"
 traits=outside small
 reverse=TileSidePigPen
 
 [TileSideCoveredBridge]
 name={ffg:TILE_COVERED_BRIDGE_MAD09}
-image="{import}/img/Tile_Covered_Bridge_MAD09.dds"
+image="{import}/img/Tile_Covered_Bridge_MAD09"
 aspect=1.5
 pps=146.28571
```

Comment:
Still don't follow how this case is being handled.

Reply:
There is a new method ContentData.ResolveTextureFile(), that resolves the name.

Reply:
The name isn't the issue, this has a different pps, which I assume will be wrong still (as with all large tiles).  This is why I was proposing alternate image file names, so that they could also have alternate pps options.  This would could also be added for all tiles rather than the default value change, but I'm not convinced on that one.

Reply:
Suggestion:

If the the file can't be identified by the filename, we add the extension, then we can do:
```
[TileSideCoveredBridge]
name={ffg:TILE_COVERED_BRIDGE_MAD09}
image="{import}/img/Tile_Covered_Bridge_MAD09.dds"
image2="{import}/img/Tile_Covered_Bridge_MAD09.pvr"
aspect=1.5
pps=146.28571
pps2=73.14285
```

Reply:
When you say "scale the texture to fit as a default" the issue is identifying these tiles.  The only different between large and small tiles is the trait meta data, the images are the same. So we could use the trait, but I don't like that idea.

Calculation is fine.  We don't even need to be that advanced, it could just be ```pps=96``` for setting absolute values and ```pps=*0.5``` for relative to default.

Also I am OK with the Image2 and pps2 solution.

Given this known bug only affects android I will merge this pull request as is for now.